### DISCUSSION TITLE
List of examples or some kind of Awesome VitePress repo

### DISCUSSION THREAD

**Discussion by davestewart** [vuejs/vitepress#1855] [Ideas]
**Is your feature request related to a problem? Please describe.**

Currently a VuePress user and excited to move to VitePress (primarily because I've written my own VP themes from scratch, and this seems to be the main design goal for v3) but:

- there are various key differences between the frameworks (bundler, plugins, config)
- there are significant intentional omissions in v3 (rather large list in the docs)
- it's still in development

As a new / migrating user, it's a bit of a minefield and difficult to know where to start when something is omitted.

**Describe the solution you'd like**

Rather than clogging up the issues with questions, it would be cool to get an Awesome VitePress list going, so users could find alternative plugins, or dig through the code of known solutions (i.e. the Vue Blog) and see how it's been solved before.

**Describe alternatives you've considered**

Clog up the issues with questions, bug @kiaking on Twitter, Google solutions, etc.

**Additional context**

Happy to help search and review resources to get it started.

**Comment by Zhengqbbb** [comment]
Looking forward to the Awesome VitePress repo 😎

**Reply by ansonchieng** [reply]
We have recently built one, feel free to check out our awesome vitepress repo here - https://github.com/logicspark/awesome-vitepress-v1 😎

We are also open to contributions, if anyone is interested please let me know :)
